Cargo.toml

```diff
@@ -50,6 +52,7 @@ bincode = { version = "0.8.0" }
 num-iter = { version = "0.1.35", default-features = false }
 criterion = { version = "0.3" }
 doc-comment = { version = "0.3" }
+pure-rust-locales = { version = "0.2.1" }
```

Comment:
I don't think that this is necessary here, if chrono fails to build without the `locales` feature then that probably means that it's not correctly configured out (missing some `#[cfg(locales)]` functionality).

Reply:
yes sorry that was a leftover, fixed